### PR TITLE
[TASK] Add more fields in table for StoreFieldsAsXmlToDbFinisher

### PR DESCRIPTION
**Pull request by esokoll2**

### CODE REVIEW

File: Configuration/TCA/tx_formtools_requests.php [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
     'ctrl' => [
         'title' => 'Stored Forms',
-        'label' => 'email',
+        'label' => 'last_name',
         'tstamp' => 'tstamp',
         'crdate' => 'crdate',
         'cruser_id' => 'cruser_id',
         'dividers2tabs' => true,
-        'default_sortby' => 'ORDER BY email',
+        'default_sortby' => 'ORDER BY tstamp',
```
Comment (froemken) [inline Configuration/TCA/tx_formtools_requests.php:10]: Perfect

File: Configuration/TCA/tx_formtools_requests.php [hunk 2]
```diff
@@ -2,12 +2,12 @@
 return [
     'ctrl' => [
         'title' => 'Stored Forms',
-        'label' => 'email',
+        'label' => 'last_name',
```
Comment (froemken) [inline Configuration/TCA/tx_formtools_requests.php:5]: Can you please add `label_alt` and `label_alt_force` to add first_name, too?

File: Configuration/TCA/tx_formtools_requests.php [hunk 3]
```diff
@@ -125,27 +125,63 @@
             'l10n_mode' => 'exclude',
             'l10n_display' => 'defaultAsReadonly'
         ],
-        'email' => [
+        'first_name' => [
             'exclude' => true,
-            'label' => 'E-Mail',
+            'label' => 'Vorname',
```
Comment (froemken) [inline Configuration/TCA/tx_formtools_requests.php:130]: Can you add localization for these labels and all below?

File: ext_tables.sql [hunk 4]
```diff
@@ -2,8 +2,12 @@
 # Table structure for table 'tx_formtools_requests'
 #
 CREATE TABLE tx_formtools_requests (
+	first_name varchar(255) DEFAULT '' NOT NULL,
+	last_name varchar(255) DEFAULT '' NOT NULL,
+	telephone varchar(255) DEFAULT '' NOT NULL,
```
Comment (froemken) [inline ext_tables.sql:7]: 30 should be enough for telephone

**Review by froemken — CHANGES_REQUESTED** [review]

File: Configuration/TCA/tx_formtools_requests.php (hunk 2 above)
Comment (esokoll2) [reply Configuration/TCA/tx_formtools_requests.php:5]: Done

File: Configuration/TCA/tx_formtools_requests.php (hunk 1 above)
Comment (esokoll2) [reply Configuration/TCA/tx_formtools_requests.php:12]: Done

File: ext_tables.sql (hunk 4 above)
Comment (esokoll2) [reply ext_tables.sql:7]: Done

Comment (esokoll2) [reply ext_tables.sql]: Done

**Review by froemken — APPROVED** [review]

**Pull request merged**